### ISSUE TITLE
New pypi release for removing debug print

### ISSUE THREAD

**Issue by andresriancho** [ionelmc/python-tblib#4]
```python
@return_error
def apply_with_return_error(args):
    """
    args is a tuple where the first argument is a callable.

    eg::

        apply_with_return_error((func, 1, 2, 3)) - this will call func(1, 2, 3)

    """
    print args[0]
```

The `print args[0]` which was removed in https://github.com/ionelmc/python-tblib/commit/a2bd19dcbc197124ef8a213501c5eb2aa1cbf231 is still in pypi, which makes this library unusable (without re-writing my own apply_with_return_error)

A new pypi release is needed

**Comment by ionelmc** [comment]
Oooops. Released `0.2.0` now on pypi.

**Comment by ionelmc** [comment]
Thanks for reporting this.
